rpc/Load/1.2.0/RPCLoad.ts

```diff
@@ -0,0 +1,320 @@
+
+    @RPCMethod({
+        require: ['shipmentUuid', 'senderWallet', 'moderatorWallet'],
+        validate: {
+            uuid: ['shipmentUuid', 'senderWallet', 'moderatorWallet'],
+        },
+    })
+    public static async SetModeratorTx(args) {
+        const senderWallet = await Wallet.getById(args.senderWallet);
+        const moderatorWallet = await Wallet.getById(args.moderatorWallet);
+
+        const LOAD_CONTRACT: LoadContract = <LoadContract>loadedContracts.get(PROJECT, VERSION);
+
+        const txUnsigned = await LOAD_CONTRACT.setModeratorTx(senderWallet, args.shipmentUuid, moderatorWallet.address);
+
+        return {
+            success: true,
+            transaction: txUnsigned,
+        };
+    }
+
+    @RPCMethod({
+        require: ['shipmentUuid', 'senderWallet'],
+        validate: {
+            uuid: ['shipmentUuid', 'senderWallet'],
+        },
+    })
+    public static async SetInProgressTx(args) {
+        const senderWallet = await Wallet.getById(args.senderWallet);
+
+        const LOAD_CONTRACT: LoadContract = <LoadContract>loadedContracts.get(PROJECT, VERSION);
+
+        const txUnsigned = await LOAD_CONTRACT.setInProgressTx(senderWallet, args.shipmentUuid);
+
+        return {
+            success: true,
+            transaction: txUnsigned,
+        };
+    }
+
+    @RPCMethod({
+        require: ['shipmentUuid', 'senderWallet'],
+        validate: {
+            uuid: ['shipmentUuid', 'senderWallet'],
+        },
+    })
+    public static async SetCompleteTx(args) {
+        const senderWallet = await Wallet.getById(args.senderWallet);
+
+        const LOAD_CONTRACT: LoadContract = <LoadContract>loadedContracts.get(PROJECT, VERSION);
+
+        const txUnsigned = await LOAD_CONTRACT.setCompleteTx(senderWallet, args.shipmentUuid);
+
+        return {
+            success: true,
+            transaction: txUnsigned,
+        };
+    }
+
+    @RPCMethod({
+        require: ['shipmentUuid', 'senderWallet'],
+        validate: {
+            uuid: ['shipmentUuid', 'senderWallet'],
+        },
+    })
+    public static async SetCanceledTx(args) {
+        const senderWallet = await Wallet.getById(args.senderWallet);
+
+        const LOAD_CONTRACT: LoadContract = <LoadContract>loadedContracts.get(PROJECT, VERSION);
+
+        const txUnsigned = await LOAD_CONTRACT.setCanceledTx(senderWallet, args.shipmentUuid);
+
+        return {
+            success: true,
+            transaction: txUnsigned,
+        };
+    }
+
+    @RPCMethod({
+        require: ['shipmentUuid', 'senderWallet', 'depositAmount'],
+        validate: {
+            uuid: ['shipmentUuid', 'senderWallet'],
+        },
+    })
+    public static async FundEscrowTx(args) {
+        const senderWallet = await Wallet.getById(args.senderWallet);
+
+        const LOAD_CONTRACT: LoadContract = <LoadContract>loadedContracts.get(PROJECT, VERSION);
+        const TOKEN_CONTRACT: ShipTokenContract = <ShipTokenContract>loadedContracts.get('ShipToken');
```

Comment:
My intuition is that the `'ShipToken'` string should be pulled out into a constant but I'm not sure where it should live. @mlclay?